src/server/services/documents.js

```diff
@@ -0,0 +1,36 @@
+const debug = require('debug')('persist-uploads')
```

Comment:
should probably be `documents`. we should also maybe namespace. if we do `require('debug')('arpa.documents')` then we can run `DEBUG='arpa.*' yarn serve` and get all of our debug output

Reply:
Oh cool.  That's a good idea.